CHANGELOG.md

```diff
@@ -1,11 +1,14 @@
 # Change Log
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 10.6.8
+_June 22, 2023_
+- Add new icon type `scan-id`.  [PR #432](https://github.com/saddlebackdev/react-cm-ui/pull/441)
```

Comment:
We also have to add an entry about my bug fix too (from #440).
I usually do this separately.  Like I start release branch, bump the version number in `package.json` and `versions.js` (`npm install` so `package-lock.json` also has no version, and do this in a commit in release branch, then finish the release and publish the package.
See https://github.com/saddlebackdev/react-cm-ui/wiki/Deployment-And-Publishing#step-by-step

Reply:
Add a new entry for PR #440 in the CHANGELOG

Reply:
Looks good.  Resolving.